src/dialog/dialog.directive.ts

```diff
@@ -114,6 +114,19 @@ export class DialogDirective implements OnInit, OnDestroy, OnChanges {
 		this.toggle();
 	}
 
+	@HostListener("keydown", ["$event"])
+	onKeydown(event: KeyboardEvent) {
```

Comment:
There's already a block of observables dealing with this in `ngOnInit` ... It'd be great if we could co-locate all this event handling code ... either merging this into the `ngOnInit` block, or moving that functionality to this style.